extensions/wikia/ImageReview/ImageReviewHelper.class.php

```diff
@@ -151,30 +183,30 @@ public function refetchImageListByTimestamp( $timestamp ) {
 
 		$db = $this->getDatawareDB( DB_SLAVE );
 
-		// try to re-fetch the previuos set of images
+		// try to re-fetch the previous set of images
 		// TODO: optimize it, so we don't do it on every request
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)